pytorch_caney/model/pipelines/modis_segmentation.py

```diff
@@ -362,7 +303,7 @@ def main(hparams: Namespace):
     # ------------------------
     trainer = Trainer(
         accelerator="gpu",
-        devices=8,
+        devices=7,
```

Comment:
Maybe we can add a cli argument with the default being torch.cuda.device_count(), so you would end up with args.gpu_count or something. and just override it from the CLI if we need to.

Reply:
Added in the latest commit. Thanks!